### PR TITLE
JENA-1490: Handle blank nodes client-server when using Fuseki.

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

**Comment by afs** [comment]
Note: WIP at this time.

There are tests to be tidied up and put into the PR.

**Comment by afs** [comment]
Final changes now made.

**Comment by afs** [comment]
Add controllable parse-check of SPARQL request strings.

File: jena-rdfconnection/src/main/java/org/apache/jena/rdfconnection/RDFConnectionRemoteBuilder.java [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+        requireNonNull(txnLifecycle);
+        requireNonNull(destination);
+        
+        Function<RDFConnectionRemoteBuilder, RDFConnection> maker = creator ;
+        
+        if ( maker == null )
+            maker = (b)->b.buildConnection();
+        
+        // Sort out service URLs.
+        // Delay until here because the order of destination and service settign sisnot
```
Comment (rvesse) [inline jena-rdfconnection/src/main/java/org/apache/jena/rdfconnection/RDFConnectionRemoteBuilder.java]: Typo - `setting sisnot` -> `settings is not`

**Pull request merged**